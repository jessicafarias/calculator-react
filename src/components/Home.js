const Home = () => (
  <div className="bg-image3">
    <div className="div-center mt-2">
      <h1>Welcome to my website</h1>
      <p>
        {' '}
        In this page you can make some math calculations.
        This calculator project has the basic operations like sum,
        multiplication or change the sign of the number.
      </p>
    </div>
  </div>
);

export default Home;
