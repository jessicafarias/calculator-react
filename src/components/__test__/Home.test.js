import { BrowserRouter } from 'react-router-dom';
import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer'; // ES6
import Home from '../Home';

describe('Display Home page', () => {
  it('renders correctly', () => {
    const tree = TestRenderer.create(<BrowserRouter><Home /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
