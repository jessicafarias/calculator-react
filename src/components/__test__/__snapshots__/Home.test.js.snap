// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Home page renders correctly 1`] = `
<div
  className="bg-image3"
>
  <div
    className="div-center mt-2"
  >
    <h1>
      Welcome to my website
    </h1>
    <p>
       
      In this page you can make some math calculations. This calculator project has the basic operations like sum, multiplication or change the sign of the number.
    </p>
  </div>
</div>
`;
